src/Apps/Order/routes.tsx

```diff
@@ -22,6 +22,29 @@ import { ShippingProps } from "./Routes/Shipping"
 // @ts-ignore
 import { SubmissionProps } from "./Routes/Submission"
 
+/**
+ * Used to match against child route locations. Supports order or order2.
+ * Captures the child route name in the first capture group.
+ *
+ * https://regexper.com/#%5C%2Forder%5Cd%3F%5C%2F.%2B%5C%2F%28%5Cw%2B%29
+ */
+const ORDER_REGEX = /\/order\d?\/.+\/(\w+)/
+
+const confirmRouteExit = location => {
+  // Refresh
+  if (window && location.pathname === window.location.pathname) {
```

Comment:
This should be `typeof window !== 'undefined'`; I believe `window` as a value will error out.